e2e/scenario/scenario_ubuntu2204-airgap.go

```diff
@@ -0,0 +1,22 @@
+package scenario
+
+import (
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+)
+
+func (t *Template) ubuntu2204AirGap() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204-airgap",
+		Description: "Tests that a node using the Ubuntu 2204 VHD and is airgap can be properly bootstrapped",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
```

Comment:
yeah, ideally we would have airgap cluster selectors/mutators - a good follow-up item

Reply:
can you create a work item for this?

Reply:
https://dev.azure.com/msazure/CloudNativeCompute/_workitems/edit/27242881